.changeset/purple-dots-refuse.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Persist styles of persistent client:only components during view transitions
```

Comment:
```suggestion
Allows styles of `client:only` components to persist during view transitions
```
I can see this is a draft, but also that there's only going to be a changeset here, so I'll just give some pre-advice now then duck out!

For the final version, imagining the context in which these are read, it's often helpful to think of this statement as the answer to the question, "What does this PR do?" That starts with an implied "This PR..."

So, lots of things are possible! I just left one option! But if people are scanning changelogs, it's helpful to know instantly whether something was a fix, vs something new added (to use/learn about), vs a change in behaviour... So another alternative could be something like:

> Fixes styles of `client:only` components not persisting during view transitions

Lots of options! I'll just remind you that describing a particular code behaviour like the original statement isn't the same as telling someone, "This is what this PR gives you/does for you now" while priming them for whether this is a fix to a problem they might have had, or something new to explore, etc.

Reply:
Thank you very much, Sarah, I really appreciate your suggestions!